pkg/app/app.go

```diff
@@ -31,20 +31,22 @@ import (
 func defaultFuncOrganizationSlug(engine workflow.Engine, config configuration.Configuration, logger *zerolog.Logger, apiClientFactory func(url string, client *http.Client) api.ApiClient) configuration.DefaultValueFunction {
 	err := config.AddKeyDependency(configuration.ORGANIZATION_SLUG, configuration.ORGANIZATION)
 	if err != nil {
-		logger.Print("Failed to add dependency for ORGANIZATION_SLUG:", err)
+		logger.Err(err).Msg("Failed to add dependency for ORGANIZATION_SLUG on ORGANIZATION")
```

Comment:
I would minimize changes, so not change the log messages if not really necessary.